tools/typings/q/Q.d.ts

```diff
@@ -0,0 +1,335 @@
+// Type definitions for Q
```

Comment:
How about taking the "Typings" folder out of the "src" folder and putting besides src? That way we will not have to duplicate the typings?